docs/application/dotnet/guides/multimedia/media-controller.md

```diff
@@ -112,7 +110,52 @@ To send a command from the client and to process it on the server side:
     }
     ```
 
-<a name="servermetadata"></a>
+## Updating and Retreiving Playlist
```

Comment:
Please correct the spelling as ‘Retrieving’.

Reply:
Done